js/ngDialog.js

```diff
@@ -7,7 +7,7 @@
 (function (root, factory) {
     if (typeof module !== 'undefined' && module.exports) {
         // CommonJS
-        module.exports = factory(require('angular'));
+        module.exports = require('angular');
```

Comment:
@DevinClark I'm not often work with angular right now, but it seems the problem is in it:

```javascript
var angular = require('angular')
```

This will bring you `undefined`..